.travis.yml

```diff
@@ -1,38 +1,51 @@
 ---
-language: node_js
-node_js:
-  - "6"
+jobs:
+  include:
+    - stage: lint
+      language: python
+      python: 3.6
+      before_install:
+        - export PATH=$PATH:./node_modules/.bin
+        - export NODE_PATH=$NODE_PATH:./node_modules
+        - nvm install
+        - npm install
+        - npm install eslint remark remark-lint remark-cli
```

Comment:
why not add this to `package.json` (via `yarn add`) ?

Reply:
Also might be good the pin the version, as specified in coala-bears [package.json`](https://github.com/coala/coala-bears/blob/master/package.json)

Reply:
Alright

Reply:
Note that eslint in coala-bears is v3

But v4 should work, as [gci-leaders is using it](https://github.com/coala/gci-leaders/blob/master/package.json#L51)